code/src/components/StyledComponents.js

```diff
@@ -0,0 +1,165 @@
+import styled from 'styled-components'
+
+
+export const TasksContainer = styled.section`
+display: flex;
+flex-direction: column;
+box-sizing: border-box;
+padding: 20px;
+width: 100%;
+height: auto;
+font-family: 'Poppins', sans-serif;
+`
+
+export const TaskContainer = styled.div`
+display: flex;
+flex-direction: row;
+align-items: center;
+justify-content: space-between;
+box-sizing: border-box;
+width: 100%;
+height: auto;
+margin: 10px 0;
+
+`
+export const Todo = styled.label`
+display: flex;
+position: relative;
+flex-direction: row;
+align-items: center;
+justify-content: space-between;
+box-sizing: border-box;
+width: 87%;
+min-height: 55px; 
+margin: 0; 
+padding: 15px 10px;
+background-color: #EFEDFF;
+border-radius: 15px;
+`
+
+export const RemoveButton = styled.button`
+border-radius:50%;
+width: 25px;
+height: 25px;
+border: 2px solid lightgrey;
+background-color: white;
+color: lightgrey;
+font-weight: bold;
+cursor: pointer;
+`
+
+export const TaskTitle = styled.p`
+margin: 0;
+margin-left: 35px;
+color: ${props => props.checked ? "#D1C4F6" : "#7F2AEC"};
+text-decoration: ${props => props.checked ? "line-through" : "none"};
+font-weight: bold;
+` 
+export const TimeStamp = styled.p`
+position: absolute;
+bottom: 0;
+right: 10px;
+color: #D1C4F6;
+font-size: 10px;
+text-align: right;
+` 
+const now = new Date()
```

Comment:
Although one could argue that "now" is a "styled date element" , this now variable seems a little bit lost here ;)